drivers/ethernet/oa_tc6.c

```diff
@@ -256,6 +256,11 @@ int oa_tc6_check_status(struct oa_tc6 *tc6)
 
 static void oa_tc6_update_status(struct oa_tc6 *tc6, uint32_t ftr)
 {
+	if (oa_tc6_get_parity(ftr)) {
+		LOG_ERR("OA Status Update: Footer parity error!");
```

Comment:
This could easily flood logger if we receive lot of packets with parity errors. Would DBG level be enough here if you also update relevant ethernet statistics. See `include/zephyr/net/net_stats.h` and `net_stats_eth_errors` struct if there is some variable we could update here.

Reply:
This error shall not happen too often - if it does - then there is some issue with the transmission medium. To "reuse" statistics this function shall return error code when the parity error is detected (respective values are going to be updated latter i the code).